Assets/ShooterProject/Scripts/Actors/Health/Health.cs

```diff
@@ -52,6 +52,9 @@ private set
 		public event Action<Health> OnDied;
 		public event Action<Health> OnChanged;
 
+		public event Action onDamaged;
+		public event Action onHealed;
```

Comment:
```suggestion
		public event Action OnDamaged;
		public event Action OnHealed;
```
public поля и свойства, namespace'ы: PascalCase;